projects/core/src/cart/cart.module.ts

```diff
@@ -32,6 +34,7 @@ export class CartModule {
     return {
       ngModule: CartModule,
       providers: [
+        provideDefaultConfig(defaultCartConfig),
```

Comment:
Is it necessary to provide defaultConfig in this case?

Reply:
default config removed. No need for it really in that simple case.